test/model/finders_test.rb

```diff
@@ -37,6 +37,17 @@ class Human
     test 'return nil when the id does not belong to an object of this bucket' do
       assert_equal nil, Human['not_an_id']
     end
+
+    test 'retrieve an object with given quorum' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.